DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -127,5 +148,31 @@ public int GetNumberOfActiveCoursesAtCentreForCategory(int centreId, int adminCa
                 new { centreId, adminCategoryId }
             );
         }
+
+        public IEnumerable<CourseStatistics> GetCourseStatisticsAtCentreForCategoryId(int centreId, int categoryId)
+        {
+            return connection.Query<CourseStatistics>(
+                @$"SELECT
+                        cu.CustomisationID,
+                        cu.CentreID,
+                        cu.Active,
+                        cu.AllCentres,
+                        ap.ASPMenu,

```

Comment:
Should we still pull this?

Reply:
Good spot, forgot to get rid of it here. Will do that now.